pkg/config/msp.go

```diff
@@ -95,9 +94,10 @@ type KeyInfo struct {
 	// the case of Software BCCSP as well as the HSM BCCSP would be
 	// the SKI of the key.
 	KeyIdentifier string
-	// KeyMaterial (optional) for the key to be imported; this is
-	// properly encoded key bytes, prefixed by the type of the key.
-	KeyMaterial *ecdsa.PrivateKey
+	// KeyMaterial (optional) for the key to be imported; this
+	// must be a supported PKCS#8 private key type of either
+	// *rsa.PrivateKey, *ecdsa.PrivateKey, or ed25519.PrivateKey.
+	KeyMaterial interface{}
```

Comment:
Andy why isn’t this a crypto.PrivateKey ?

Reply:
wasn't aware there was a type for that, I just used what the `x509.MarshalPKCS8PrivateKey` accepted. I'll update it